libr/core/agraph.c

```diff
@@ -4939,6 +4939,7 @@ R_API RAGraph *create_agraph_from_graph(const RGraph/*<RGraphNodeInfo>*/ *graph)
 	if (!result_agraph) {
 		return NULL;
 	}
+	result_agraph->need_reload_nodes = false;
```

Comment:
Why was this required?

Reply:
TLDR: `need_reload_nodes` causes agraph to attempt loading either callgraph or basic block graph, which doesn't make sense in this case. 

Because agraph is mixing multiple responsibilities within single module. On hand it deals with task of making exact drawing on other it also handles reloading of data from r2 for some kinds of graphs like function CFG. Many of agraph properties are specific to one kind of graph.

Reply:
ok! ;)